packages/platform-android/src/link/patches/__tests__/applyParams.test.ts

```diff
@@ -9,12 +9,12 @@ import applyParams from '../applyParams';
 
 describe('applyParams', () => {
   it('apply params to the string', () => {
-    expect(applyParams('${foo}', {foo: 'foo'}, 'react-native')).toEqual(
+    expect(applyParams('${foo}', {foo: 'foo'} as any, 'react-native')).toEqual(
       'getResources().getString(R.string.reactNative_foo)',
     );
   });
 
   it('use null if no params provided', () => {
-    expect(applyParams('${foo}', {}, 'react-native')).toEqual('null');
+    expect(applyParams('${foo}', {}, 'react-native')).toEqual('');
```

Comment:
why this changed?

Reply:
Oh, my bad. I have changed the logic of `applyParams` function and forgot to revert it back.